docs/get-started/getting-started-gcp.md

```diff
@@ -22,7 +22,10 @@ gcloud services enable storage-api.googleapis.com
 ``` 
 
 After the command line environment is setup, log in to your GCP account.  You can now create a Dataproc cluster with the configuration shown below.
-The configuration will allow users to run any of the [notebook demos](../demo/GCP) on GCP.  Alternatively, users can also start 2*2T4 worker nodes.
+The configuration will allow users to run any of the [notebook demos](https://github.com/NVIDIA/spark-rapids/tree/branch-0.2/docs/demo/GCP) on GCP.  Alternatively, users can also start 2*2T4 worker nodes.
```

Comment:
Why did we make the link explicit to branch-0.2? It fells like it is adding in maintenance to maintain this link, especially because we don't consistently do it for all of the demos.

Reply:
The ../demo/GCP folder does not render on gh-pages (perhaps because there are only notebook files in the folder).  The main branch does not have this folder.  I linked to 0.2 for now.  When we update main and it has the demo/GCP folder, I can update the link there.  

Related links: 
https://nvidia.github.io/spark-rapids/docs/get-started/getting-started-gcp.html 
https://nvidia.github.io/spark-rapids/docs/demo/GCP
https://github.com/NVIDIA/spark-rapids/tree/main/docs